src/pages/Dashboard.vue

```diff
@@ -102,8 +103,8 @@ export default defineComponent({
   },
   methods: {
     async testClient() {
-      this.global.client?.do?.runTest()
-      this.global.client?.data?.farming.events.on("farmedBlock", this.farmBlock)
+      this.client?.do?.blockSubscription.runTest()
+      this.client?.data?.farming.events.on("farmedBlock", this.farmBlock)
```

Comment:
Same here, all `?` should be removed, none of the fields are optional as far as I see.